fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/InsertIntoTableCommand.java

```diff
@@ -166,7 +175,13 @@ public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
         state.addTableIndexes(physicalOlapTableSink.getTargetTable());
 
         executor.setProfileType(ProfileType.LOAD);
+        executor.setPlanner(planner);
```

Comment:
why set planner here?

Reply:
moved to `generateStreamLoadNereidsPlan` function